composer.json

```diff
@@ -5,7 +5,7 @@
     "homepage": "https://airbrake.io",
     "license": "MIT",
     "require": {
-        "php": ">=5.4"
+        "php": "^5.4.7 || ^7.0"
```

Comment:
Okay, then I think we should use `5.5.9` since lower versions of `5.5` is as bad as `5.4`. 

Do you mind if I update this PR with the new version?